src/Modules/Station/Components/OnlineUsers/OnlineUsers.tsx

```diff
@@ -14,55 +16,17 @@ import {
 import { DEFAULT_USER_AVATAR } from '../../../User/Constants';
 import './OnlineUsers.scss';
 
-const fixture = [
-  {
-    id: '5acdce8373f8d20004bc3314',
-    name: 'Mars',
-    username: 'lybaokhanh',
-    avatarUrl: '',
-    points: 200,
-  },
-  {
-    id: '5acdce8373f8d20004bc3314',
-    name: 'Lamth2',
-    username: 'lamth2',
-    avatarUrl: '',
-    points: 130,
-  },
-  {
-    id: '5acdce8373f8d20004bc3314',
-    name: 'Liquid',
-    username: 'lybaokhanh',
-    avatarUrl: '',
-    points: 455,
-  },
-  {
-    id: '5acdce8373f8d20004bc3314',
-    name: 'Navi',
-    username: 'lybaokhanh',
-    avatarUrl: '',
-    points: 600,
-  },
-];
-
-const currentUser = {
-  id: '5acdce8373f8d20004bc3314',
-  name: 'Lamth2',
-  username: 'lamth2',
-  avatarUrl: '',
-  points: 10,
-  password: '',
-  email: 'lamth2@gmail.com',
-};
-
 interface IProps {
   userInfo?: RegisteredUser;
+  data: any;
```

Comment:
data does not make sense. I suggest listOnlineUser.
Do we have Type for data. Should not use any